exampleSite/content/about-split/index.md

```diff
@@ -4,11 +4,11 @@ lastmod = "2017-10-05"
 date = "2017-10-05"
 subtitle = ""
 layout = "split"
-sidebarTitle = "Sidebar"
+sidebar_title = "Sidebar"
 sidebar = """
-So much information  
-Phone numbers  
-Details  
+So much information
```

Comment:
The whitespace was intentional to force new lines in markdown ;)

Reply:
Yeah my editor settings remove whitespaces by default now.